pkg/ansible/watches/watches_test.go

```
@@ -107,9 +107,7 @@ func TestLoad(t *testing.T) {
 		ValidRole:     filepath.Join(cwd, "testdata", "roles", "role"),
 	}
 
-	tmpl, err := template.ParseFiles("testdata/valid.yaml.tmpl")
-	if err != nil {
-	}
+	tmpl, _ := template.ParseFiles("testdata/valid.yaml.tmpl")
```

instead of ignoring the error, I would fail the test.

```
if err != nil {
   t.Fatalf("Unable to parse template: %v", err)
```